src/systems/ec/implementations/react/packages/media-container/stories/Index.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import demoContent from '@ecl/ec-specs-media-container/demo/data';
+
+import MediaContainer from '../MediaContainer';
+
+storiesOf('MediaContainer', module)
+  .addDecorator(withKnobs)
+  .add('video', () => (
+    <MediaContainer
+      description={text('Description', demoContent.description)}
+      image={text('Image', demoContent.image)}
+      sources={demoContent.sources}
```

Comment:
it is possible to have more complex knobs (see cards for instance), but this can be improved later